src/test/java/com/amazon/ion/impl/bin/IonRawBinaryWriterTest_1_1.kt

```diff
@@ -933,13 +1047,13 @@ class IonRawBinaryWriterTest_1_1 {
     }
 
     @Test
-    fun `write an e-expression`() {
+    fun `write a delimited e-expression`() {
```

Comment:
🗺️ They're not really "delimited", but I'm using this term as the opposite of "length-prefixed".